src/views/CVView/CVView.tsx

```diff
@@ -2,50 +2,92 @@ import { UserCV } from "../../components/UserCV/UserCV";
 //import { UserCard } from "../../components/UserCard/UserCard";
 import { IoIosArrowDown } from "react-icons/io";
 import { Header } from "../../components/Header/Header";
-import React from "react";
+import React, {useEffect, useState} from "react";
 
 import "./CVView.scss";
 import "../../index.scss"
 
 export const CVView = () => {
+    const contractType = ['', 'Umowa o pracę', 'B2B', 'Umowa zlecenie', 'Umowa o dzieło'];
+    const id = "46f84261-df9d-11ed-a2b7-24fd5235b3db"; //@TODO zamienić na prawdziwe ID
+    const [mail, setMail] = useState("");
+    const [data, setData] = useState({
+        bio:"",
+        bonusProjectUrls : "",
+        canTakeApprenticeship : 1,
+        courseCompletion : 1,
+        courseEngagement : 1,
+        courses : "",
+        education : "",
+        expectedContractType : 1,
+        expectedSalary : 0,
+        expectedTypeWork : 1,
+        firstName : "",
+        githubUsername : "",
+        lastName : "",
+        monthsOfCommercialExp : 0,
+        phoneNumber : "",
+        portfolioUrls : "",
+        projectDegree : 1,
+        projectUrls : "",
+        targetWorkCity : "",
+        teamProjectDegree : 1,
+        userStatus : 1,
+        workExperience : "",
+    });
+
+    useEffect( () => {
+        const fetchData = async () => {
+            const res = await fetch(`http://localhost:3001/student/getcv/${id}`);
+            const downloadData = (await res.json())[0];
+            setData(downloadData);
+            console.log(data.portfolioUrls);
+            const resMail = await fetch(`http://localhost:3001/user/getemail/${id}`);
+            const email = (await resMail.json());
+            setMail(email);
+        }
+        fetchData()
+            .catch(console.error);
+    }, []);
+
   return (
     <div className="CVView__container">
       <Header/>
-      <div className="page__container">
```

Comment:
Zmieniasz strukturę tss, a scss nie jest zmieniony. testowałeś to czy się widok nie rozwali? Ten page__container spełniał istotną funkcję, strzałka 'wróć' musi być poza głównym kontenerem.

Reply:
Faktycznie, poprawiłem